raiden/transfer/balance_proof.py

```diff
@@ -66,6 +70,43 @@ def pack_signing_data(
     return data_that_was_signed
 
 
+def pack_signing_data2(
+        nonce,
+        balance_hash,
+        additional_hash,
+        channel_identifier,
+        token_network_identifier,
+        chain_id,
+) -> bytes:
+    balance_proof = raiden_libs.messages.BalanceProof(
+        channel_identifier=channel_identifier,
+        token_network_address=token_network_identifier,
+        balance_hash=balance_hash,
+        nonce=nonce,
+        additional_hash=additional_hash,
+        chain_id=chain_id,
+    )
+
+    return balance_proof.serialize_bin()
+
+
+def signing_update_data(
+        balance_proof,  # BalanceProofSignedState
```

Comment:
Why not use the classname as the type here and have it only as coment? Cyclic dependencies?

Reply:
Yes, cyclic dependencies.

Reply:
Hmm I see. Will try to fix them soon. I also got hit at some point by them.